gravatar/src/main/java/com/gravatar/extensions/UserProfileExtensions.kt

```diff
@@ -3,15 +3,53 @@ package com.gravatar.extensions
 import com.gravatar.AvatarQueryOptions
 import com.gravatar.AvatarUrl
 import com.gravatar.ProfileUrl
-import com.gravatar.api.models.GalleryImage
-import com.gravatar.api.models.Link
-import com.gravatar.api.models.Profile
-import com.gravatar.api.models.ProfileContactInfo
-import com.gravatar.api.models.ProfilePayments
-import com.gravatar.api.models.VerifiedAccount
+import com.gravatar.restapi.models.GalleryImage
+import com.gravatar.restapi.models.Link
+import com.gravatar.restapi.models.Profile
+import com.gravatar.restapi.models.ProfileContactInfo
+import com.gravatar.restapi.models.ProfilePayments
+import com.gravatar.restapi.models.VerifiedAccount
 import com.gravatar.types.Hash
 import java.net.URI
 import java.time.Instant
+import com.gravatar.api.models.GalleryImage as LegacyGalleryImage
+import com.gravatar.api.models.Link as LegacyLink
+import com.gravatar.api.models.Profile as LegacyProfile
+import com.gravatar.api.models.ProfileContactInfo as LegacyProfileContactInfo
+import com.gravatar.api.models.ProfilePayments as LegacyProfilePayments
+import com.gravatar.api.models.VerifiedAccount as LegacyVerifiedAccount
+
+/**
+ * Get the hash for a user profile.
+ */
+public fun LegacyProfile.hash(): Hash {
```

Comment:
>However, the IDE clearly shows when you are using the deprecated class.

I only thought about this because IDE won't show a warning on something like this `legacyProfile.hash()` unless the `LegacyProfile.hash()` is deprecated. I'm ok with leaving it as is.